xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxAccountService.java

```diff
@@ -21,4 +24,19 @@ public AccountInfo getSubaccountInfo(String subaccount) throws IOException {
     return FtxAdapters.adaptAccountInfo(
         getFtxAccountInformation(subaccount), getFtxWalletBalances(subaccount));
   }
+
+  @Override
+  public AccountLeverageSetting setInitialLeverage(AccountLeverageSetting setting) throws IOException {
+      try {
+        ftx.changeLeverage(
+                exchange.getExchangeSpecification().getApiKey(),
+                exchange.getNonceFactory().createValue(),
+                signatureCreator,
+                null,
```

Comment:
On FTX we use sub-accounts.

Please also pass the subaccount parameter to the API.